docs/src/components/ConsentManager.js

```diff
@@ -1,9 +1,36 @@
 import React from 'react'
-import { ConsentManager, openConsentManager } from '@segment/consent-manager'
-import inEU from '@segment/in-eu'
+import inRegions from '@segment/in-regions'
+import { ConsentManager, CloseBehavior } from '@segment/consent-manager'
 import segmentWriteKey from '../../segmentWriteKey'
 
 export default () => {
+  const inCA = inRegions(['CA'])
+  const inEU = inRegions(['EU'])
+  const shouldRequireConsent = inRegions(['CA', 'EU'])
+  const caDefaultPreferences = {
+    advertising: false,
+    marketingAndAnalytics: true,
+    functional: true
+  }
+
+  const euDefaultPreferences = {
```

Comment:
I'm wondering if it makes more sense for the consent manager itself to own the defaults for given regions? Or maybe you can pass a `isCCPA` that handles the default preferences?

Reply:
I say this because it seems a bit cumbersome to put the onus on consumers to know which default preferences to use for which regions.

Reply:
That's a really good idea!!  I wanted to be careful about that from a legal standpoint, and thought putting ownership on consumers would release us from any liability around guaranteeing CCPA compliance.

Maybe we can do that in a next version if we get it approved by legal/ leave an asterisk?